### ISSUE TITLE
Shield-transform option

### ISSUE THREAD

**Issue by JoinGitHub23** [CartoDB/mobile-sdk#390]
Hello! 
In carto exist option "shield-transform" (https://carto.com/developers/styling/cartocss/#shield-transform-functions), which defines SVG transformation functions to scale how shields appear. But in mobile-sdk this option not supported. So, as far as I can tell, there is no way to change the image size for shields. And this is very inconveniently. Is it possible to add a zoom function to the raster image shields? 
Thanks

**Comment by mtehver** [comment]
That is correct, 'shield-transform' is not currently supported. The list of supported CartoCSS features and extensions is listed here: https://github.com/CartoDB/mobile-sdk/wiki/CartoCSS-notes

I will tag this with 'idea-maybe-future'.
